app/javascript/components/student/tracks-list/TagsFilter.jsx

```diff
@@ -1,21 +1,33 @@
-import React, { useState } from 'react'
+import React, { useState, useEffect, useRef } from 'react'
 import { TagOptionList } from './TagOptionList'
 
 export function TagsFilter({ options, dispatch }) {
-  const [hidden, setHidden] = useState(true)
+  const [expanded, setExpanded] = useState(false)
```

Comment:
It's better to use positive words for this stuff than negative ones. Otherwise you get negating of negation (not hidden) which hurts my brain :)